### ISSUE TITLE
Load yolov8 model 

### ISSUE THREAD

**Issue by Dimlight** [ultralytics/ultralytics#1639]
### Search before asking

- [X] I have searched the YOLOv8 [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

I wanted to load yolov8 using the following code snippet. It failed: 

Code: 

import torch
import cv2 as cv
from torchvision.ops import nms

path = '../test2.png'

model = torch.hub.load('ultralytics/ultralytics','custom', path='../Object_Detection/yolov8.pt',force_reload=True,source='local', pretrained =False)

img = cv.imread(path)

result = model(cv.cvtColor(img, cv.COLOR_BGR2RGB))
# print(result)

df = result.pandas().xyxy[0]
print(df)
for ind in df.index:
    x1, y1 = int(df['xmin'][ind]), int(df['ymin'][ind])
    x2, y2 = int(df['xmax'][ind]), int(df['ymax'][ind])
    label = df['name'][ind]
    cv.rectangle(img,(x1,y1), (x2,y2), (255,255,0),2)
    cv.putText(img,label,(x1,y1-5), cv.FONT_HERSHEY_PLAIN, 2,(255,255,0),2)
cv.imshow('IMAGE',img)
cv.waitKey(0)

Error:

File "Projects/TestingCodesEkram/Object_Detection/detect_simple.py", line 13, in <module>
    model = torch.hub.load('ultralytics/ultralytics','custom', path='../Object_Detection/yolov8.pt',force_reload=True,source='local', pretrained =False)
  File "/Projects/TIPS/temp/.venv/lib/python3.10/site-packages/torch/hub.py", line 542, in load
    model = _load_local(repo_or_dir, model, *args, **kwargs)
  File "/Projects/TIPS/temp/.venv/lib/python3.10/site-packages/torch/hub.py", line 569, in _load_local
    hub_module = _import_module(MODULE_HUBCONF, hubconf_path)
  File "/Projects/TIPS/temp/.venv/lib/python3.10/site-packages/torch/hub.py", line 90, in _import_module
    spec.loader.exec_module(module)
  File "<frozen importlib._bootstrap_external>", line 879, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1016, in get_code
  File "<frozen importlib._bootstrap_external>", line 1073, in get_data

**Comment by Dimlight** [comment]
Note: I used custom dataset to train the model.

**Comment by glenn-jocher** [comment]
@Dimlight your usage is incorrect. See https://docs.ultralytics.com/modes/predict for correct Usage examples.

**Comment by Dimlight** [comment]
I have updated the code in the following way: 

import torch
import cv2 as cv
from torchvision.ops import nms

from ultralytics import YOLO 


path = '/home/ehossai2/Projects/TestingCodesEkram/Object_Detection/test2.png'

model = YOLO("../Object_Detection/yolov8.pt")

img = cv.imread(path)

result = model(cv.cvtColor(img, cv.COLOR_BGR2RGB))

print(result)

Can you please help me to understand how to extract label names and bounding box information?

**Comment by glenn-jocher** [comment]
@Dimlight you can print(result) to see the structure and access whatever you want, i.e.

```python
result.boxes.xywh
result.boxes.cls
```

See https://docs.ultralytics.com/modes/predict for details

**Comment by pderrenger** [comment]
@Gertje79 Yes, you can indeed iterate through the results using a for loop. Each `box` object will contain the bounding box information and label for each detected object. This allows you to access the individual bounding boxes and labels for further processing or visualization.
